<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupInfoForm</class>
 <widget class="QDialog" name="GroupInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="Main_frame">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background-color: rgb(30, 30, 30);
	border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="Titile_bar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="Title_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="Title_label">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font-family: 'Roboto Regular';
font-size: 20px;</string>
              </property>
              <property name="text">
               <string>Информация</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="Btn_frame">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border: none;
	border-radius: 8px;
	background-color: rgb(230, 0, 0);
}

QPushButton:hover
{
	border: none;
	border-radius: 8px;
	background-color: rgb(200, 0, 0);
}

QPushButton:pressed
{
	border: none;
	border-radius: 8px;
	background-color: rgb(160, 0, 0);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Content_bar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="GroupName_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font-family: 'Roboto Regular';
font-size: 16px;</string>
              </property>
              <property name="text">
               <string>Название комнаты</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MaxNumOfUsers_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font-family: 'Roboto Regular';
font-size: 16px;</string>
              </property>
              <property name="text">
               <string>Максимальное количество пользователей: 100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="CurrNumOfUsers_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font-family: 'Roboto Regular';
font-size: 16px;</string>
              </property>
              <property name="text">
               <string>Текущее количество участников</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="CurrUsers_label">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font-family: 'Roboto Regular';
font-size: 16px;</string>
                </property>
                <property name="text">
                 <string>Текущие участники:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="UsersWidget">
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>180</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QListWidget
{
	background-color: rgb(35, 35 , 35);
}

QListWidget::item
{
	border: 1px solid rgb(45, 45 , 45);
	border-radius: 5px;
	padding-left: 5px;
	padding-right: 5px;
	background-color: rgb(40, 40 , 40);
}

QListWidget::item:hover
{
	border: 1px solid rgb(77, 230, 230);
}

QListWidget::item:selected
{
	color:rgb(77, 230, 230);
	/*color:rgb(0, 180, 255);*/
	border: 1px solid rgb(77, 230, 230);
	background-color: rgb(45, 45 , 45);
}</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Button_bar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>273</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	border: 2px solid rgb(38, 38, 38);
	border-radius:  15px;
	font-family: 'Roboto Regular';
	font-size: 16px;
	background-color: rgb(35, 35 , 35);
}
QPushButton::hover
{
	border: 2px solid rgb(214, 0, 0);
	background-color: rgb(45, 45, 45);
}
QPushButton::pressed
{
	border: 2px solid rgb(140, 0, 0);
	background-color: rgb(35, 35 , 35);
}</string>
           </property>
           <property name="text">
            <string>Закрыть</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
